apps/s_client.c

```diff
@@ -708,10 +718,14 @@ const OPTIONS s_client_options[] = {
     {"enable_pha", OPT_ENABLE_PHA, '-', "Enable post-handshake-authentication"},
     {"enable_server_rpk", OPT_ENABLE_SERVER_RPK, '-', "Enable raw public keys (RFC7250) from the server"},
     {"enable_client_rpk", OPT_ENABLE_CLIENT_RPK, '-', "Enable raw public keys (RFC7250) from the client"},
-#ifndef OPENSSL_NO_SRTP
+# ifndef OPENSSL_NO_ECH
+    {"ech_config_list", OPT_ECHCONFIGLIST, 's',
+     "Set ECHConfigList, value is base 64 encoded ECHConfigList"},
+# endif
+# ifndef OPENSSL_NO_SRTP
     {"use_srtp", OPT_USE_SRTP, 's',
      "Offer SRTP key management with a colon-separated profile list"},
-#endif
+# endif
```

Comment:
I wouldn't change the lines not relevant to ech.

Reply:
ah, I didn't know you had a magic want to turn down the style checker! will revert that so